sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java

```diff
@@ -86,7 +86,7 @@ public void testBulkRead() throws Exception {
   @Test
   public void testWriteReadUsingDefaultAppendMethod() throws Exception {
     ArrayList<KV<String, String>> data = new ArrayList<>();
-    for (int i = 0; i < 100; i++) {
+    for (int i = 0; i < 8000; i++) {
```

Comment:
If the test had used this value instead, unit tests would have detected the issue.